packages/ui/src/components/kitchen-sink/_kitchenSink.scss

```diff
@@ -1,13 +1,20 @@
 .root {
+  position: absolute;
+  top: 0;
+  left: 0;
+  overflow: hidden;
   width: 100%;
   height: 100%;
-  background: rgba(0, 0, 0, .7);
+  transition: opacity 500ms;
+  opacity: 0;
 }
 
-/* TODO should fix in player and remove */
-:global(.playkit-side-panel-content) {
-  & > div {
-    width: 100%;
-    height: 100%;
-  }
+.active {
```

Comment:
move this into `.root` to improve specification for that one. consider
```
.root {

   &.active {
  }
}

Reply:
done